### DISCUSSION TITLE
Does Ocelot Gateway support SSE (Server-Sent Events)?

### DISCUSSION THREAD

**Discussion by fikrifdhl1** [ThreeMammals/Ocelot#1775] [Q&A] [answered]
Hi everyone, I need help and suggestions. I have a back-end service that provides SSE for web clients. My plan is our web client will access SSE via ocelot Gateway. But as far as I know, Ocelot only supports web socket and SignalR. So I want to try make my custome handler to support SSE. Can you guys give me a hint or some little guide to achieve this?

I'd really appreciate any suggestions.
Thank you

**Comment by raman-m** [comment, marked as answer]
Hello!
If you mean this
- [Server-sent events - Web APIs | MDN](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events)
- [Using server-sent events - Web APIs | MDN](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events)
- [HTML SSE API](https://www.w3schools.com/html/html5_serversentevents.asp)

No, we don't support this technology officially. But you can try to write a routes and see either HTTP traffic goes through Ocelot app or not.
It depends on transport protocol. Ocelot supports: HTTP(S). Websocket (SignalR) transport protocols.
You can research and let us know your testing results.

**Comment by raman-m** [comment]
@fikrifdhl1 Soon... 👇 
- #2383
